.github/workflows/test-PRs.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test PRs
```

Comment:
Name is a bit odd, should be something more generic if we're testing all branches